<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DualCalculatorWidget</class>
 <widget class="QWidget" name="DualCalculatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="name">
     <property name="placeholderText">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="close">
     <property name="icon">
      <iconset theme="edit-delete">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="firstClass"/>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="firstClassLevel">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="secondClass"/>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="secondClassLevel">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
